distributed/worker_memory.py

```diff
@@ -265,7 +267,23 @@ async def _maybe_spill(self, worker: Worker, memory: int) -> None:
                 self._throttled_gc.collect()
                 memory = worker.monitor.get_process_memory()
 
-        self._maybe_pause_or_unpause(worker, memory)
+            now = monotonic()
+
+            # Increase spilling aggressiveness when the fast buffer is filled with a lot
+            # of small values. This artificially chokes the rest of the event loop -
+            # namely, the reception of new data from other workers.
+            # DO NOT tweak this without thorough stress testing.
+            # See https://github.com/dask/distributed/issues/6110.
```

Comment:
Add a note for future readers that this event loop blocking is a bit of a hack? I'm assuming in the future, we might properly pause during eviction once we have https://github.com/dask/distributed/issues/5702?

Reply:
That's closed by #6195 :)

Reply:
Added note